src/main/java/com/battleship/backend/controllers/GameController.java

```diff
@@ -49,21 +49,27 @@ Boardable placeShip(@RequestBody Request placeRequest) throws Exception{
         }
     }
 
+    // change to return message instead of board.
     @PatchMapping("/board/hit")
     public @ResponseBody
-    Boardable[] hitShip(@RequestBody Request hitRequest) throws Exception{
+    String hitShip(@RequestBody Request hitRequest) throws Exception{
```

Comment:
Inconsistent spacing with the spaces before and after brackets and parens in some places. A linter would help with this in the future.

Reply:
Fixed the spacing after the Exceptions.